scripts/imm_utils.sh

```diff
@@ -18,7 +18,7 @@
 IMM_CONFIG_PATH=~/.immconfig
 
 get_token() {
-	user=$1
+	[[ -n $DEFAULT_TENANT_NAME ]] && user="admin" || user=$1
```

Comment:
i'd revert to `user=$1` since this function should get a specified user's token, we could just call get_token with proper user name. i.e.
we could add 2 variables:
`ADMIN_NAME=admin`
`[[ -n $DEFAULT_TENANT_NAME ]] && USER_NAME=admin || USER_NAME=user`
and then use them in test_imm